app/pages/docs/image-optimization.mdx

```diff
@@ -1,76 +1,81 @@
-10 images or 10 million images.
+In plaats van afbeeldingen te optimaliseren op build time, doet Blitz dit
+on-demand, wanneer gebruikers ze opvragen. In tegenstelling tot bij
+statische sitegenerators en andere statische oplossingen, worden je build
+times niet verhoogd, of je nu 10 of 10 miljoen afbeeldingen gebruikt.
 
-Images are lazy loaded by default. That means your page speed isn't
-penalized for images outside the viewport. Images load as they are
-scrolled into viewport.
+Afbeeldingen worden standaard _lazy loaded_. Dat wil zeggen dat je
+paginasnelheid niet wordt verminderd door afbeeldingen die buiten de
+viewport vallen. Afbeeldingen worden geladen wanneer ze in de viewport
+gescrolled worden.
 
-Images are always rendered in such a way as to avoid
-[Cumulative Layout Shift](https://web.dev/cls/), a
-[Core Web Vital](https://web.dev/vitals/) that Google is going to
-[use in search ranking](https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html).
+Afbeeldingen worden altijd op zo'n manier weergegeven dat ze
+[Cumulative Layout Shift](https://web.dev/cls/), een
+[Core Web Vital](https://web.dev/vitals/) die Google gaat
+[gebruiken in de ranking van zoekresultaten](https://webmasters.googleblog.com/2020/05/evaluating-page-experience.html),
+vermijden.
 
-## Image Component {#image-component}
+## Afbeeldingscomponent {#image-component}
 
-To add an image to your application, import the `Image` component:
+Om een afbeelding aan je applicatie toe te voegen, importeer je de `Image`
+component:
 
 ```jsx
 import {Image} from "blitz"
 
 function Home() {
   return (
     <>
-      <h1>My Homepage</h1>
+      <h1>Mijn homepagina</h1>
       <Image
-        src="/me.png"
-        alt="Picture of the author"
+        src="/ik.png"
+        alt="Afbeelding van de auteur"
         width={500}
         height={500}
       />
-      <p>Welcome to my homepage!</p>
+      <p>Welkom op mijn homepagina!</p>
     </>
   )
 }
 
 export default Home
 ```
 
-## Configuration {#configuration}
+## Configuratie {#configuration}
 
-In addition to using properties available to the `Image` component, you
-can optionally configure Image Optimization for more advanced use cases
-via `blitz.config.js`.
+Naast het gebruik van de properties beschikbaar in de `Image` component,
+kan je optioneel afbeeldingsoptimalisatie configureren voor geavanceerdere
+use cases via `blitz.config.js`.
 
-### Domains {#domains}
+### Domeinen {#domains}
 
-To enable Image Optimization for images hosted on an external website, use
-an absolute url for the Image `src` and specify which `domains` are
-allowed to be optimized. This is needed to ensure that external urls can't
-be abused.
+Om afbeeldingsoptimalisatie in te schakelen voor afbeeldingen die gehost
+worden op een externe website, gebruik je een absolute URL voor de `src`
+van Image en specifieer je welke `domains` toegestaan zijn om te
```

Comment:
```
van `Image` en specifieer je welke `domains` toegestaan zijn om te
```
Heeft in de originele docs ook geen quotes. Maar daar gebruiken ze die wel op alle andere plaatsen dus gaat meer om consistentie.